### DISCUSSION TITLE
How to display the AxisLabelText in the LocalRenderView?

### DISCUSSION THREAD

**Discussion by spermatozoa** [Kitware/trame#235] [Q&A] [answered]
Hi,
I recently tested the example [trame-vtk/examples/validation/AxesActor.py](https://github.com/Kitware/trame-vtk/blob/master/examples/validation/AxesActor.py) and noticed that the axis labels (e.g., "x", "y", "z") seem to disappear in the LocalRenderView. Despite using axes.AxisLabelsOn(), there doesn't seem to be any effect in the LocalRenderView. 
Is there a way to display the AxisLabelText in the LocalRenderView?

**Comment by jourdain** [comment, marked as answer]
Labels would need to be implemented in vtk.js to have that part working.

**Reply by spermatozoa** [reply]
Ok, thanks!
